snap/snapcraft.yaml

```diff
@@ -0,0 +1,145 @@
+name: sdc
+version: HEAD
+summary: A sane D compiler with an LLVM backend
+description: |
+    SDC is a from-scratch implementation of a D compiler written itself
+    in the D programming language, while using LLVM as its backend.  It
+    is currently under heavy development, and hence provides a limited
+    feature set that is not yet suitable for production work.
+
+    A major project goal is to provide a D compiler as a library (libd)
+    so as to support the development of a much wider range of D tools.
+
+confinement: classic
+grade: devel
+
+apps:
+  sdc:
+    command: bin/sdc
+
+parts:
+  sdc:
+    source: .
+    source-commit: HEAD
+    source-type: git
+    plugin: make
+    make-parameters:
+    - LLVM_CONFIG=../../../stage/bin/llvm-config
+    - DFLAGS=-m64 -w -debug -g -unittest
+    - LDFLAGS=-L../../../stage/lib
+    artifacts:
+    - bin
+    - import
```

Comment:
Should the `import` directory actually be included in the final package?  (this covers `import/java` and `import/llvm`).